src/EPR.SubsidiaryBulkUpload.Application/Services/BulkUploadOrchestration.cs

```diff
@@ -1,21 +1,29 @@
 ﻿using EPR.SubsidiaryBulkUpload.Application.DTOs;
+using EPR.SubsidiaryBulkUpload.Application.Models;
+using EPR.SubsidiaryBulkUpload.Application.Services.Interfaces;
 
 namespace EPR.SubsidiaryBulkUpload.Application.Services;
 public class BulkUploadOrchestration : IBulkUploadOrchestration
 {
+    private const string SubsidiaryBulkUploadProgress = "Subsidiary bulk upload progress";
+
     private readonly IRecordExtraction recordExtraction;
     private readonly ISubsidiaryService organisationService;
     private readonly IBulkSubsidiaryProcessor childProcessor;
+    private readonly INotificationService _notificationService;
 
-    public BulkUploadOrchestration(IRecordExtraction recordExtraction, ISubsidiaryService organisationService, IBulkSubsidiaryProcessor childProcessor)
+    public BulkUploadOrchestration(IRecordExtraction recordExtraction, ISubsidiaryService organisationService, IBulkSubsidiaryProcessor childProcessor, INotificationService notificationService)
     {
         this.recordExtraction = recordExtraction;
         this.organisationService = organisationService;
         this.childProcessor = childProcessor;
+        _notificationService = notificationService;
     }
 
-    public async Task Orchestrate(IEnumerable<CompaniesHouseCompany> data, Guid userId)
+    public async Task Orchestrate(IEnumerable<CompaniesHouseCompany> data, UserRequestModel userRequestModel)
     {
+        _notificationService.SetStatus($"{userRequestModel.UserId}{userRequestModel.OrganisationId}{SubsidiaryBulkUploadProgress}", "Uploading");
```

Comment:
Use an extension method to get the key so it can be consistently applied in other services

Reply:
Done.